### DISCUSSION TITLE
Google Sheet를 디비로 쓰는건 어떠세요?

### DISCUSSION THREAD

**Discussion by sangjuun-park** [LifeIsAlone/public-home#20] [Ideas]
이제 점점 느려지네요
그래서 GeoCoder의 과부화를 줄이기위해 DB를 찾아보는데 
Google Sheet가 있네요, 재민님 데이터 전처리 하실때도 엑셀시트 많이 쓰시니까 편하게
Google Sheet를 DB로 써서 한번 운영해보시는건 어떠세요?

https://developers.google.com/sheets/api/quickstart/js

저가 한번 실험해보고 말씀드릴까요? 재민님

**Comment by cyberjam** [comment]
대박 따봉입니다 상준님! 
처음엔 기존 그대로 가볍게 가자는 생각으로 어떠한 이유 때문에 구글 스프레드시트를 굳이 도입해야하는가 열심히 토론했던게 기억나네요.

이번에 사용자 입장에서 매입임대주택 청약할때 사용해보니까 구글 스프레드 시트에서 관리자? 또는 사용자가 import를 해서 네이버 또는 카카오 지도에 띄워주는 게 좋겠다 생각했습니다.

마치 "구글 내지도" 처럼요. 
초반에는 제가 데이터 전처리해서 올리긴 할테지만 나중에는 아영이네아빠께도 사용법을 알려드려 피드백을 받아봐도 좋겠습니다. 

참고로 저는 개발당시 js로 전처리를.. 정말 python 마렵더라구요. 그리고 무엇보다 이렇게 적은 데이터는 스프레드시트로 관리하는게 한눈에 볼 수 있어 좋기도 하구요. 

구글 스프레드시트 도입 적극 찬성입니다!
next.js 이전 작업이나 지금 제 프로젝트 마무리 되는대로 함께 재밌는 이야기 또 해봐요 우리

**Reply by cyberjam** [reply]
구글 내지도와 차이가 있다면 K-지도라 전철과 주변 편의시설을 볼 수 있다는 점

**Comment by cyberjam** [comment]
기록 : 참고로 속도가 느려진 이슈는 변하지 않는 고정된 데이터이므로 json에 위경도를 미리 박아놓았습니다.
